### PR TITLE
Western Sahara ION rival template

### PR DESCRIPTION
**Pull request by ElmexH**
## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:
An version of the Western Sahara ION faction as Rivals that doesn't depend on Aegis. Uses WS and vanilla content and if enabled also content from other DLC's. 

In my opinion ION adds nice variation compared to other rival factions with ION fielding more modern equipment and vehicles but lacking in heavier vehicles and body armor. 

I think the faction works as an nice outlet to add some of the more exotic Western Sahara and Reaction Forces content to Antistasi that wouldn't necessarily fit the Vanilla factions. 

Although Aegis has some really nice additions it's compatibility with other mods (such as ACE) can be a bit problematic . A version of ION not dependent of Aegis could be a nice addition to the existing templates. 

Apart from Western Sahara itself only other CDLC content used is from Reaction Forces (if enabled)
### Please specify which Issue this PR Resolves (If Applicable).
N/A

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

The faction appears to load without any errors. 
Thus far I've not been able to figure out an reasonable way to get the factions units to spawn in game to validate that everything works as intended and seems balanced. 

### How can the changes be tested?
Steps:
Load WS CDLC (+ RF)
Start a new save with ION WS as rivals
Start looking for Rival faction?
********************************************************
Notes: Any feedback would be appreciated. 
Might implement this as an extender if you're not interested in merging this, but I think it would fit in nicely with the existing templates.

### CODE REVIEW

**Comment by Silence-112** [comment]
When making a PR, select unstable. We don't push straight to stable (for obvious reasons)

**Comment by ElmexH** [comment]
> When making a PR, select unstable. We don't push straight to stable (for obvious reasons)

That was the intention, but forgot to apparently actually set it. Fixed now

**Review by wersal454 — COMMENTED** [review]
"B_UAV_06_F", "B_UAV_06_medical_F"

No real point in adding this 2, but wouldn't make a difference since they don't really drop their own bombs, though I might add functionality to "B_UAV_06_F" like uavs to drop 
flyers

**Review by wersal454 — COMMENTED** [review]
O_LSV_02_armed_F not sure about CSAT LSV, maybe use NATO one? Or even both since they have black camos

**Comment by wersal454** [comment]
If I remember correctly, RF adds ION pickups as well, so I would suggest looking into that, there probably both HMG and MMG variants. If there isn't any classnames just use variants system
https://community.bistudio.com/wiki/Reaction_Forces/classNames

Also can try Apex 4x4 since it has black camo as well

**Comment by wersal454** [comment]
Other than that, looks fine

**Comment by ElmexH** [comment]
Removed the mentioned drones and added the missing pickups. 

Chose the Qilin instead of the Prowler due an better matching color in my opinion and it adding more weapon variety with the minigun. I think its not unreasonable to assume ION would have other CSAT equipment as they have for example the Marids. 

I also don't think the 4x4s would necessarily be a great fit visually even with the black color scheme.

I think the Prowlers and the 4x4s don't necessarily add that much capability wise to the vehicle lineup and adding them would mean the offroads and pickups would have less of an presence. 

Would it make any sense to add the ION AA pickup in the lighArmedVehicles?

**Comment by wersal454** [comment]
Rivals don't use good AA, only stuff like anti lowair mgs. You can experiment with tanks, putting ATGM variant of NATO apc in it, or regular APC(if it has ION camo)
image



Also checked LSVs collor pallet look pretty identical to me to be honest, but your choose
image
image

Also remove ["I_LT_01_cannon_F", ["Indep_Olive",1]], if you don't use this tank. It won't break anything, but who knows

**Comment by wersal454** [comment]
I'm adding more ION vehicles, so might wanna wait a bit ,or merge it and come back to later

**Comment by Silence-112** [comment]
> I'm adding more ION vehicles, so might wanna wait a bit ,or merge it and come back to later

Adding more ION vehicles... to what?

**Comment by wersal454** [comment]
> > I'm adding more ION vehicles, so might wanna wait a bit ,or merge it and come back to later
> 
> Adding more ION vehicles... to what?

to config fixes

**Comment by wersal454** [comment]
Better idea, just copy config fixes from my PR
https://github.com/SilenceIsFatto/A3-Antistasi-Ultimate/pull/160/commits/94461897e9411545d7650c9a2e7db88800d08970

File: A3A/addons/core/Templates/Templates/WS/WS_Riv_ION.sqf [hunk 1]
```diff
@@ -0,0 +1,959 @@
...
+	];
+};
+
+
+/////Vests
+
+private _vests = ["V_Chestrig_blk", "V_Chestrig_oli", "V_TacVest_blk", "V_TacVest_oli", "V_TacVestIR_blk", "V_lxWS_TacVestIR_oli"];
+
+if (_hasRF) then {
+	_vests append ["V_TacVest_blk", "V_TacVest_oli"]; //TODO: Incorrect items?
```
Comment (Silence-112) [inline A3A/addons/core/Templates/Templates/WS/WS_Riv_ION.sqf:405]: I assume this still needs to be done?

**Pull request merged**